src/session/content/message_row/indicators_model.rs

```diff
@@ -35,6 +36,13 @@ mod imp {
                         glib::Object::static_type(),
                         glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY,
                     ),
+                    glib::ParamSpecBoolean::new(
+                        "is-edited",
+                        "Is edited",
+                        "Whether the message of the model was edited",
+                        false,
+                        glib::ParamFlags::READABLE,
+                    ),
```

Comment:
I think you can just use the `message-info` property. This way you don't even need the new label inside the `MessageIndicators`.